### PR TITLE
Dependency injection task completed

### PR DESCRIPTION
**Pull request by Kgaur624**
[https://confluence.dev.lithium.com/display/CD/Lab%3A+Dependency+Injection](Lab C111)
In this task I have finished dependency injection using spring

### CODE REVIEW

File: projecty/src/main/java/com/resource/TweetController.java [hunk 1]
```diff
@@ -1,56 +1,60 @@
...
             } else {
-                logger.error("Request incomplete");
-                return Response.status(500, "Request incomplete").build();
+                Status status = services.sendTweets(tweet);
+                if (status.getText().equals(tweet)) {
+                    logger.info("Tweet posted Successfully");
+                    return new SendResponse("Tweet posted Successfully",200);
+                } else {
+                    logger.error("Request incomplete");
+                    return new SendResponse("Request incomplete",500);
```
Comment (anasuya1117) [inline projecty/src/main/java/com/resource/TweetController.java:38]: https://khoros.slack.com/archives/C02JGBKPMKK/p1638184739007200

please apply this

File: projecty/test/testCases/FetchTweetsTest.java [hunk 2]
```diff
@@ -116,10 +114,17 @@ public void testCase_fetchFilterTweet_successCase() throws TwitterException{
...
     }
 
 
+
+
+
+
+
+
+
```
Comment (anasuya1117) [inline projecty/test/testCases/FetchTweetsTest.java:128]: remove this

**Review by anasuya1117 — APPROVED** [review]
Address the comments and merge

File: projecty/test/testCases/FetchTweetsTest.java (hunk 2 above)
Comment (Kgaur624) [reply projecty/test/testCases/FetchTweetsTest.java:127]: I have removed

File: projecty/src/main/java/com/resource/TweetController.java (hunk 1 above)
Comment (Kgaur624) [reply projecty/src/main/java/com/resource/TweetController.java]: I have Applied

File: projecty/src/main/java/com/service/Services.java [hunk 3]
```diff
@@ -1,54 +1,52 @@
...
+        configurationBuilder=(ConfigurationBuilder)context.getBean("Configuration");
+       twitterFactory = new TwitterFactory(configurationBuilder.build());
         twitter = twitterFactory.getInstance();
     }
 
 
     public Status sendTweets(String args) throws NullPointerException, TwitterException {
+        List<SendResponse> list = new ArrayList<>();
+        String message;
+        String family;
```
[the remark this answers is not in the document]
Comment (Kgaur624) [reply projecty/src/main/java/com/service/Services.java:49]: I have removed

**Comment by Kgaur624** [comment]
I have done all the changes 
now code and test cases working fine

**Pull request merged**